vllm/worker/model_runner.py

```diff
@@ -677,6 +677,7 @@ def __init__(
             int(self.cache_config.cpu_offload_gb * 1024**3))
 
     def load_model(self) -> None:
+        logger.info("Start loading model...")
```

Comment:
Suggest clearer wording:
```suggestion
        logger.info("Starting to load model %s...", model_config.name)
```

Reply:
fixed in [bfe2bc8](https://github.com/vllm-project/vllm/pull/6670/commits/bfe2bc89c4d9e74596b73ce0368bc9b281765016)